docs/api/datetime.md

```diff
@@ -107,9 +114,15 @@ Ionic Framework makes use of the [Intl.DatetimeFormat](https://developer.mozilla
 
 ### Custom Locale
 
-For instances where you need a specific locale, you can use the `locale` property to set it.
+For instances where you need a specific locale, you can use the `locale` property to set it. The locale controls both the language and the date and time formats that are displayed.
 
-TODO
+The following example shows how to set the locale to Spanish (Spain).
+
+<CustomLocale />
+
+:::note
```

Comment:
Can we explore if adding another line-break here, would create spacing between the playground example and the note?

![Screen Shot 2022-05-02 at 12 47 48 PM](https://user-images.githubusercontent.com/13732623/166289773-71daa5ef-984e-43b4-b6ae-42e611ec1873.png)

Or, seeing this with text below the playground example as well; maybe we want to add bottom padding to the playground component?

Reply:
Updated the playground styles to add a margin:

![image](https://user-images.githubusercontent.com/2721089/166303444-597e1404-8525-4dd6-a219-ea0d9f120100.png)